src/commands/projects-addfirebase.ts

```diff
@@ -0,0 +1,23 @@
+import * as _ from "lodash";
+
+import * as Command from "../command";
+import { FirebaseError } from "../error";
+import { addFirebaseToCloudProjectAndLog, promptAvailableProjectId } from "../management/projects";
+import * as requireAuth from "../requireAuth";
+
+module.exports = new Command("projects:addfirebase [projectId]")
+  .description("add Firebase resources to a Google Cloud Platform project")
+  .before(requireAuth)
+  .action(
+    async (projectId: string | undefined, options: any): Promise<any> => {
```

Comment:
This returns `Promise<FirebaseProjectMetadata>`, doesn't it?

Reply:
Updated other places as well